src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -500,6 +500,26 @@ def get_kube_proxy_config(self) -> Union[Dict, ContainerServiceNetworkProfileKub
         # this parameter does not need validation
         return kube_proxy_config
 
+    def get_node_os_upgrade_channel(self) -> Union[str, None]:
+        """Obtain the value of node_os_upgrade_channel.
+        :return: string or None
+        """
+        # read the original value passed by the command
+        node_os_upgrade_channel = self.raw_param.get("node_os_upgrade_channel")
+
+        # In create mode, try to read the property value corresponding to the parameter from the `mc` object.
```

Comment:
I believe this is right because I saw others doing it but I didn't understand why create wouldn't it always be empty in create......

Reply:
I was curious as well and asked Fuming here:
https://github.com/Azure/azure-cli-extensions/pull/5751#discussion_r1072896380